### DISCUSSION TITLE
Riders missing in nearby rider list

### DISCUSSION THREAD

**Discussion by dfkdg** [SauceLLC/sauce4zwift#82] [General]
Just tried Sauce$Zwift. In the Nearby Riders window, the riders seemed to be in order of distance in front and behind me as per the rider list on Zwift, yet quite a few riders were missing on Sauce. This is on an older Macbook Pro.

**Comment by mayfield** [comment]
Were they possibly stationary or going the opposite direction?  I have stricter filters than what Zwift themselves do.

**Reply by dfkdg** [reply]
No I could see on the Zwift rider list there were people in front of me and they were not showing on the Sauce nearby rider list - at least as far as I could see, and not in the correct place if they were displayed somewhere else. Of say the first 6 people displayed in front of me on the Zwift rider list, perhaps only 2 or 3 would be shown on the Sauce list. Of those shown on the Sauce list the times were within a second of those shown on the Zwift list.

**Reply by dfkdg** [reply]
And no they were not stationary, these were riders all going the same way as me on the Zwift list.

**Reply by dfkdg** [reply]
I could try it again and see if it is still the same.

**Reply by dfkdg** [reply]
Tried it again and fiddled with columns and settings and it seems to be fine now sorry for the false alarm. Not sure what the problem was. Have it set up now nicely with transparent background and looks good. Will be quite useful as a bunch sweep. Thanks for you time.

**Reply by mayfield** [reply]
No worries, glad it's working.  The implementation for the groups and nearby riders are completely unique between Zwift and Sauce, so it's not too unexpected for there to be some differences.  In general I try to error on the side of showing less noise, i.e. riders far away, opposite direction, not moving, etc.

**Reply by PerBacks** [reply]
I found that one could sort the list based on names, position etc to get the sorting one wanted. I added the column position and sort on that column as ascending order
